spec/forms/schools/on_boarding/current_step_spec.rb

```diff
@@ -471,7 +473,8 @@
       let :school_profile do
         FactoryBot.create :school_profile,
           :with_dbs_requirement,
-          :with_candidate_requirement,
```

Comment:
There's a lot of the same `FactoryBot.create :school_profile` usage

Presumably this could be abstracted to a single trait, or inherited factory?

Then other later changes to these screens would reduce the volume of repeated changes?